docs/docs/dev_docs/dapps/tutorials/main.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Dapp Development Tutorials
+---
+
+- Links to specific tutorials
+
+- Connecting to the RPC Server
+- Creating Accounts
+- Deploying a contract
+- Contract Interaction and View methods
+- Simulating and sending transactions
+- Querying state and retrieving logs
```

Comment:
Perfectly valid suggestion. I thought 'this could all be one tutorial' and tried to break into much smaller pieces. I already merged some (believe it or not) and threw it out these for people to argue for further merging.